src/blueapi/service/main.py

```diff
@@ -49,6 +73,13 @@ async def on_key_error_404(_: Request, __: KeyError):
     )
 
 
+@app.put("/reload")
+def update_task(handler: BlueskyHandler = Depends(get_handler)):
```

Comment:
This seems to cause a silent crash after 1-5 reloads (random), can you reproduce?

Reply:
From discussion:
- Could switch subprocess to spawning rather than forking

Reply:
I haven't been able to reproduce the reloading issue.

Have switched to spawn now.